sjsonnet/test/src-jvm-native/sjsonnet/ErrorTests.scala

```diff
@@ -170,12 +184,24 @@ object ErrorTests extends TestSuite{
         |    at [Import].(sjsonnet/test/resources/test_suite/error.import_folder_slash.jsonnet:17:1)
         |""".stripMargin
     )
-    "import_static-check-failure" - check(
-      """sjsonnet.StaticError: Unknown variable: x
-        |    at [Id x].(sjsonnet/test/resources/test_suite/lib/static_check_failure.jsonnet:2:1)
-        |    at [Import].(sjsonnet/test/resources/test_suite/error.import_static-check-failure.jsonnet:1:1)
-        |""".stripMargin
-    )
+    "import_static-check-failure" - {
+      check(
+        """sjsonnet.StaticError: Unknown variable: x
+          |    at [Id x].(sjsonnet/test/resources/test_suite/lib/static_check_failure.jsonnet:2:1)
+          |    at [Import].(sjsonnet/test/resources/test_suite/error.import_static-check-failure.jsonnet:1:1)
+          |""".stripMargin
+      )
+      check(
+        """[warning] sjsonnet.StaticError: Unknown variable: x
+          |    at [Id x].(sjsonnet/test/resources/test_suite/lib/static_check_failure.jsonnet:2:1)
+          |
+          |sjsonnet.Error: Unknown variable: x
+          |    at [Id x].(sjsonnet/test/resources/test_suite/lib/static_check_failure.jsonnet:2:1)
+          |    at [Import].(sjsonnet/test/resources/test_suite/error.import_static-check-failure.jsonnet:1:1)
+          |""".stripMargin,
+        noStaticErrors = true
+      )
+    }
```

Comment:
Hi Stefan, thanks for the PR. Just a question, I didn't understood why the two checks -- the original one with ` """sjsonnet.StaticError: Unknown variable: x` and new one with warning `"""[warning] sjsonnet.StaticError: Unknown variable: x` would be needed. Can you please tell about this?